src/dwn-error.ts

```diff
@@ -26,6 +26,8 @@ export class DwnServerError extends Error {
  * DWN Server error codes.
  */
 export enum DwnServerErrorCode {
+  ConnectionSubscriptionJsonRPCIdExists = 'ConnectionSubscriptionJsonRPCIdExists',
```

Comment:
Nit: style consistency, same as next line.
```suggestion
  ConnectionSubscriptionJsonRpcIdExists = 'ConnectionSubscriptionJsonRpcIdExists',
```

Reply:
Done